### PR TITLE
Fix for texture used as color and bump map

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: projectwriter.py [hunk 1]
```diff
@@ -2209,7 +2209,7 @@ def __emit_texture(self, texture, bump_bool, scene, node=None, material_name=Non
                 color_space = 'srgb'
 
             filepath = util.realpath(texture.image.filepath)
-            texture_name = texture.name + "_bump" if bump_bool is True else texture.name
+            texture_name = texture.name
```
Comment (dictoon) [inline projectwriter.py]: Do we need the `texture_name` variable then?

**Comment by jdent02** [comment]
Okay to merge this?

**Comment by dictoon** [comment]
Absolutely! Merging now.

**Comment by dictoon** [comment]
Thanks for the quick fix.

**Pull request merged**